tests/specs/test/allow_all/__test__.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  "args": "test --config ../../../config/deno.json --allow-all main.ts",
```

Comment:
My comment here wasn't exactly accurate because yeah it's sometimes useful to use deno/std. We'll fix this in the future as part of https://github.com/denoland/deno/issues/25561